### PR TITLE
Fixes v1 doc structure

### PR DESCRIPTION
**Pull request by mbleigh**
Since we reverted doc structure for v1, this should generate docs successfully though I haven't vetted thoroughly if it catches everything we need/expect.

### CODE REVIEW

File: docgen/content-sources/v1/toc.yaml [hunk 1]
```diff
@@ -1,148 +1,148 @@
 toc:
   - title: 'functions'
-    path: /docs/reference/functions/v1_cloud_functions_.html
```
Comment (egilmorez) [inline docgen/content-sources/v1/toc.yaml:3]: This is a surprise to me.  The currently live reference docs don't use this versioned path (they're just https://firebase.google.com/docs/reference/functions/cloud_functions_#cloudfunction, etc).

Also, in the past I believe we manually created this file and then generated the usable, publishable _toc.yaml file from it.  Did you devise a way to autogenerate content-sources/toc.yaml?

File: docgen/content-sources/v1/toc.yaml [hunk 2]
```diff
@@ -1,148 +1,148 @@
...
+    path: /docs/reference/functions/cloud_functions.html
```
Comment (egilmorez) [inline docgen/content-sources/v1/toc.yaml:3]: Filenames currently live on firesite still use the underscores, such as cloud_functions_#cloudfunction and config_.config.

If we're 100% sure the script will generate the files without underscore naming, this should work fine.

**Review by egilmorez — COMMENTED** [review]
Couple of questions for your mbleigh, thanks!

LMK here or offline whether there's a way I can "sync" to your PR and give this a test drive.

File: docgen/content-sources/v1/toc.yaml (hunk 1 above)
Comment (mbleigh) [reply docgen/content-sources/v1/toc.yaml:3]: No, I manually corrected this based on the output of the script in this branch.

File: docgen/content-sources/v1/toc.yaml (hunk 2 above)
Comment (mbleigh) [reply docgen/content-sources/v1/toc.yaml:3]: This was modified to match the output of the gen script.

**Review by mbleigh — COMMENTED** [review]
Assuming `origin` is the public functions SDK repo:

```
git fetch origin
git checkout -b mb-docdir origin/mb-docdir
```

**Comment by egilmorez** [comment]
Let me know when/if it's time to take another look and try another run Michael. Thanks!

**Comment by egilmorez** [comment]
Hi Michael!

Since these issues were all resolved separately, can you close/drop this PR?

Until you do, I think I'm going to keep getting a bot bug in my queue.

**Comment by taeold** [comment]
@egilmorez Think https://github.com/firebase/firebase-functions/pull/990 supersedes this PR, so I'll close it!

**Pull request closed without merging**